packages/core/admin/ee/server/src/utils/sso-lock.ts

```diff
@@ -1,10 +1,8 @@
-'use strict';
+import EE from '@strapi/strapi/dist/utils/ee';
+import { isEmpty } from 'lodash/fp';
 
-const { features } = require('@strapi/strapi/dist/utils/ee').default;
-const { isEmpty } = require('lodash/fp');
-
-const isSsoLocked = async (user) => {
-  if (!features.isEnabled('sso')) {
+export const isSsoLocked = async (user: any) => {
```

Comment:
In these utils are you deliberately exporting as a default object and as named exports?

Reply:
yes, I wonder if it's something we should avoid. But some files do import it as a default object and others as named exports, so I am following this pattern, specially for utils and validations files.